src/com/TigersIter2/areaEffects/LevelUp.java

```diff
@@ -0,0 +1,25 @@
+package com.TigersIter2.areaEffects;
+
+import com.TigersIter2.entities.Avatar;
+import com.TigersIter2.entities.Entity;
+import com.TigersIter2.stats.Stats;
+import com.TigersIter2.stats.StatsModifier;
+import com.sun.xml.internal.bind.v2.TODO;
+
+/**
+ * Created by MyMac on 3/7/16.
+ */
+public class LevelUp extends AreaEffect{
+
+
+    public void affectEntity(Entity entity){
```

Comment:
what if this is called with a non-avatar entity? For example, what if an NPC walks into the tile? This should check if the entity is an avatar before trying to do this